common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -38,15 +38,20 @@
 public class WynncraftButtonFeature extends Feature {
     private static final String GAME_SERVER = "play.wynncraft.com";
     private static final String LOBBY_SERVER = "lobby.wynncraft.com";
+    private static boolean initialBoot = false;
```

Comment:
Does this boolean mark that it is currently the initial boot, or that the initial boot has passed? Change the name to be a bit more clear please

Reply:
its called "initBootPassed" now - and it marks if the TitleScreen was "loaded?" at least once.
feel free to suggest a different name for the variable